src/layouts/Navbar.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import { TfiSearch } from "react-icons/tfi";
+import { useDispatch, useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+import { setSearched as setSearchedAction } from "../services/features/store";
+
+export default function Navbar() {
+  const [search, setSearch] = useState("");
+  const [prevSearch, setPrevSearch] = useState("");
+  const dispatch = useDispatch();
+
+  const handleSearchChange = (e) => {
+    setSearch(e.target.value);
+  };
+
+  const handleSearchClick = (e) => {
+    e.preventDefault();
+    if (search !== prevSearch) {
+      dispatch(setSearchedAction(search));
+      setPrevSearch(search);
+    }
+    setSearch("");
+  };
+  return (
+    <section className="lg:hidden">
+      <div className="shadow-md fixed left-0 right-0 top-0 w-full bg-white z-50">
+        <div className="max-w-7xl px-5   mx-auto  py-6 flex gap-10 justify-between items-center flex-wrap md:flex-col">
+          <div>
+            <Link
+              to="/"
+              className="text-3xl font-bold text-red-600 cursor-pointer"
+            >
+              News Article
+            </Link>
+          </div>
+          <div className="flex-1 relative cursor-pointer">
+            <form action="#" onSubmit={handleSearchClick}>
```

Comment:
You don't need the `action` attribute because when you prevent the default behavior, the action may not be necessary.